.gitlab-ci.yml

```diff
@@ -13,6 +13,7 @@ variables:
   DOMJUDGE_VERSION:
     value:             "M.m.pp"
     description:       "The DOMjudge version, Change this variable to 7.3.3 to release the 7.3.3 dockers. The file should be available on the domjudge.org webserver."
+  DOMJUDGE_LATEST:     "lowercase boolean"
```

Comment:
What about:
```suggestion
  DOMJUDGE_LATEST:
    value:             "true"
    description:   "Whether this is the latest release"    
```

Reply:
Is indeed better, I assume that its not problematic that we will always push latest because the default value for DOMJUDGE_VERSION is already a failsafe?